lib/pysquared/hardware/rfm9x/manager.py

```diff
@@ -58,7 +56,11 @@ def radio(self) -> RFMSPI:
     def beacon_radio_message(self, msg: Any) -> None:
```

Comment:
It works as is right now, however I think it still sending the "I am beaconing" in logs when it didn't actually beacon is a *bit* misleading. I think something like this would probably be clearer in logs:

```diff
diff --git a/lib/pysquared/hardware/rfm9x/manager.py b/lib/pysquared/hardware/rfm9x/manager.py
index 0f9b61f..63c7b2a 100644
--- a/lib/pysquared/hardware/rfm9x/manager.py
+++ b/lib/pysquared/hardware/rfm9x/manager.py
@@ -57,16 +57,14 @@ class RFM9xManager:
         """Beacon a radio message and log the result."""
         try:
             if self._is_licensed:
-                sent = self.radio.send(bytes(msg, "UTF-8"))
+                self.radio.send(bytes(msg, "UTF-8"))
+                self._log.info("I am beaconing", beacon=str(msg))
             else:
-                sent = False
-                self._log.warning("Radio is not licensed, cannot send message")
+                self._log.warning("Radio is not licensed, cannot send message", beacon=str(msg))
         except Exception as e:
             self._log.error("There was an error while beaconing", e)
             return
-
-        self._log.info("I am beaconing", beacon=str(msg), success=str(sent))
-
```

Reply:
@hrfarmer that does make sense. If you want to go ahead and push that change into the branch feel free to do so!

Reply:
```suggestion
            if not self._is_licensed:
                raise ValueError("Radio is not licensed")
            
            sent = self.radio.send(bytes(msg, "UTF-8"))
```
We can fix the issue that @hrfarmer found and reduce cyclomatic complexity with something like